### ISSUE TITLE
Gather Important Discussions to Share with RDA Community

### ISSUE THREAD

**Issue by CECSpecialistI** [uwlib-cams/MARC2RDA#430]
@GordonDunsire had an idea at the end of the 2023-06-21 meeting that many of our discussions ought to be shared with the broader RDA community. Let's discuss that, and Crystal and anyone else who's interested can begin sifting through meeting notes, discussions, issues, etc. to collect things we want to share more broadly. Maybe we could share in a separate section of the Wiki, in a communication with the RSC, or by some other means.

**Comment by sarahruska** [comment]
For RSC give directly to RSC via website, not publicly. Also give PCC feedback.

**Comment by sarahruska** [comment]
https://github.com/uwlib-cams/MARC2RDA/issues/213 - 540 mapping spreadsheet comment with feedback:

> "General note:  This field may contain conditions or terms of a lack of restrictions (e.g. a CC0 1.0 public domain declaration).  RSC should consider using different terminology for the element ""has condition or restriction of use of manifestation""  and ditto for item.  Note the LC/PCC Policy statement for archival materials includes  ""Absence of restrictions should be noted.""  --lna 6/28/2023

**Comment by sarahruska** [comment]
@CECSpecialistI - would Issues and Discussions with the most comments be a good proxy for important or difficult discussions that might be worth sharing? Somewhere around 25-30+ comments?

[Issues sorted by comments descending](https://github.com/uwlib-cams/MARC2RDA/issues?q=is%3Aissue+sort%3Acomments-desc)

![Image](https://github.com/user-attachments/assets/a54c62cd-bc36-45bd-991b-3e391bf07035)

[Discussions sorted by comments descending](https://github.com/uwlib-cams/MARC2RDA/discussions?discussions_q=sort%3Acomments-desc)

![Image](https://github.com/user-attachments/assets/880c8e82-b791-45d0-82cd-9d6e01cd1e31)

**Comment by CECSpecialistI** [comment]
I don't think so, I think the most comments is just an indication of the amount of waffling we did on a topic. I don't think the RSC wants to hear about our $5 discussion, for example, or anything about classification numbers. What do you think, @GordonDunsire @corialanus ?

**Comment by sarahruska** [comment]
Thanks for the guidance! I'll start a document in the Non-Mapping Materials on the drive as a very rough draft/outline with your suggestions. We can tackle the work with more focus later. Feel free to tag me or this issue if any ideas come up in the meantime for quick reference down the road :)

**Comment by sarahruska** [comment]
https://github.com/uwlib-cams/MARC2RDA/issues/13 - 500 mapping spreadsheet comment with feedback:

> The only superclass for these entities is RDA Entity.   It's unclear whether an entity for a Description Set could be used to apply properties whose Entity relation is unclear.  Mapping to Has note about the Manifestation is temporary pending RSC review. ; lna 2022-05-17

**Comment by lake44me** [comment]
We haven't really discussed this yet and maybe it's not a problem, but I was struck by a comment @GordonDunsire made  in a meeting recently, if I heard it right, to the effect that naturally occurring objects (part of the definition of Leader 6 = r) are not RDA Entities. But in RDA we have the glossary terms

object
A carrier type that consists of a three-dimensional artifact, a replica of an artifact, or a naturally-occurring object.

object work
A work that is realized as a tactile three-dimensional form or a three-dimensional form.

It might be a good conversion test to have a record for one run through the conversion.  I found this article about cataloging 3-D objects based on RDA  https://cdn.ymaws.com/www.cilip.org.uk/resource/collection/D44BB270-A31A-428E-AD19-2C0E0BB44433/catalogue_and_index_issue_189_moore_cataloguing_three-dimensional_objects.pdf   If UW doesn't have an example, maybe just for testing we could get onw from Fresno State library catalog:  https://csu-fresnostate.primo.exlibrisgroup.com/discovery/sourceRecord?vid=01CALS_UFR:M&docId=alma991009714859702907&recordOwner=01CALS_NETWORK

**Comment by sarahruska** [comment]
https://github.com/uwlib-cams/MARC2RDA/issues/273#issuecomment-1945800106 - for consideration for PCC feedback

> "Record what is found on the manifestation in hand, even if not confirmed by the vocabulary encoding scheme" is associated with the [option](https://access.rdatoolkit.org/en-US_ala-8429b39b-af37-30fd-9611-29a717a93b7b/div_fzz_lpq_rhb) "Use any source of information". "Use judgment in deciding if it is a valid ISSN" refers to a couple of situations:
> 
> 1. The ISSN has not yet appeared in the ISSN Portal. There can be signficant delay between assigning an ISSN, then printing it on the manifestations of issues of the serial, and creating a Portal record.
> 
> 2. As with ISBNs (which are identifiers for manifestation), publishers often add the ISSNs for all serials in the "family". For example, the ISSN for an online version can be found printed on the manifestations of issues of the print version, microfiche version, etc. The WEM-lock (see RDA paper above) means that these are all distinct serial works. A user does not want to retrieve the print version when they are looking for the online version, etc. so when recording the actual ISSN for the serial work, the cataloguer must choose the correct one.
> 
> [I think it would be helpful if the PCC policy clarified this.]

**Comment by sarahruska** [comment]
From May 14, 2025 meeting discussion on [Headings for Works and Expressions as Subjects](https://github.com/uwlib-cams/MARC2RDA/wiki/2025-Meeting-Minutes#headings-for-works-and-expressions-as-subjects-30):

> Do as related person of work, mark for community conversations, and mark as loss, since loss of specificity. Talk with SAC community about subjects and related person of subjects

**Comment by CECSpecialistI** [comment]
[Deliverables document](https://docs.google.com/document/d/1nBX-76FvsAbAOYEw5WBVVYqlbYEDHtJqFdGeFv76TDo/edit?usp=sharing)
